plugin/parser/differ/mysql/differ.go

```diff
@@ -237,13 +242,61 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 					Specs: alterTableInplaceAddConstraintSpecs,
 				})
 			}
-		default:
+		case *ast.CreateViewStmt:
+			newViewList = append(newViewList, newStmt)
+		}
+	}
+
+	var tempViewList []*ast.CreateViewStmt
+	var needRealViewList []*ast.CreateViewStmt
+	for _, needView := range newViewList {
+		viewName := needView.ViewName.Name.O
+		if newNode, ok := newViewMap[viewName]; ok {
+			if !isViewEqual(needView, newNode) {
+				// skip predifined view(like temporary view in mysqldump).
+				continue
+			}
 		}
+		oldNode, ok := oldViewMap[viewName]
+		if ok {
+			if !isViewEqual(needView, oldNode) {
+				createViewStmt := needView
+				createViewStmt.OrReplace = true
+				needRealViewList = append(needRealViewList, createViewStmt)
+			}
+			delete(oldViewMap, viewName)
```

Comment:
We need to comment on deleting items from the map because that's for deleting the view if not present in the new SDL.